.github/workflows/model_registry_test.yaml

```diff
@@ -42,3 +51,25 @@ jobs:
         echo "Waiting for all Model Registry Pods to become ready..."
         kubectl wait --for=condition=available -n kubeflow deployment/model-registry-db --timeout=600s
         kubectl wait --for=condition=available -n kubeflow deployment/model-registry-deployment --timeout=600s
+
+    - name: Dry-run KF Model Registry API directly
+      run: |
+        echo "Dry-run KF Model Registry API directly..."
+        nohup kubectl port-forward svc/model-registry-service -n kubeflow 8081:8080 &
+        while ! curl localhost:8081; do echo "waiting for port-forwarding 8081"; sleep 1; done; echo "port-forwarding 8181 ready"
+        curl -v -X 'GET' \
+          'http://localhost:8081/api/model_registry/v1alpha3/registered_models?pageSize=100&orderBy=ID&sortOrder=DESC' \
+          -H 'accept: application/json'
+
+    # for these steps below ensure same steps as kserve (ie: Istio with ext auth, cert0manager, knative) so to achieve same setup 
+    - name: Port forward Istio gateway
+      run: |
+        INGRESS_GATEWAY_SERVICE=$(kubectl get svc --namespace istio-system --selector="app=istio-ingressgateway" --output jsonpath='{.items[0].metadata.name}')
+        nohup kubectl port-forward --namespace istio-system svc/${INGRESS_GATEWAY_SERVICE} 8080:80 &
+        while ! curl localhost:8080; do echo waiting for port-forwarding; sleep 1; done; echo port-forwarding ready
+
+    - name: Dry-run KF Model Registry REST API
+      run: |
+        echo "Dry-run KF Model Registry REST API..."
+        export KF_TOKEN="$(kubectl -n default create token default)"
```

Comment:
We should probably user a proper kubeflow user namespace instead of just "default"

Reply:
this is exactly per KServe test:

https://github.com/kubeflow/manifests/blob/0805068fbcc5f0cde54f1a5009d8ddbba6a8e1c3/.github/workflows/kserve_m2m_test.yaml#L73

Reply:
Probably something we should fix there as well. But that is for another PR @lampajr Ill approve for now.